docs/migrations/web/MIGRATION-v3.md

```diff
@@ -6,12 +6,173 @@ Introducing version 3 of the _spirit-web_ package.
 
 ## Overview
 
+- [General Changes](#general-changes)
+  - [Themes](#themes)
+  - [Spacing Values](#spacing-values)
+  - [Helpers: Typography and Headings](#helpers-typography-and-headings)
+  - [Utilities: Removed and New Classes](#utilities-removed-and-new-classes)
+  - [Migrating Your Own Components](#migrating-your-own-components)
 - [Component Changes](#component-changes)
-  - [Button: renamed modifier `square`](#button-square-modifier-renamed-to-symmetrical)
+  - [Button: `square` Modifier Renamed to `symmetrical`](#button-square-modifier-renamed-to-symmetrical)
+
+## General Changes
+
+### Themes
+
+Starting from version 3, the design system now supports multiple themes based on the
+theme tokens.
+
+See our [Themes decision][themes-decision] for more information and why we decided to implement this feature.
+
+#### Introduction
+
+The design system now supports multiple themes based on the theme tokens. As in
+[Figma][spirit-figma-file], where the first theme column is the default, the first theme in the
+tokens map `@themes` is the default.
+
+We use CSS variables to apply colors to the components. This way, we can easily switch
+the theme by adding a CSS class to the root element. This change affects all nested components
+that apply any color.
+
+The default theme is set to the `:root` element and all the themes are generated
+to CSS classes that can be applied to the any element. Their names are based on the
+theme names in tokens.
```

Comment:
I still think this can be more straightforward:

1. introduce CSS variables on the root element, nothing about switching
2. introduce CSS classes that can be applied on any element